.gitignore

```diff
@@ -39,7 +39,7 @@ pt_out.csv
 reg_memb.txt
 tar_memb.txt
 gt_panda*.txt
-
+*~
```

Comment:
Hey @katehoffshutta what is this supposed to do?

Reply:
This ignores my emacs backup files and doesn't need to be there for others. I will remove it